### PR TITLE
(GH-153) Add documentation about extension package

### PR DESCRIPTION
**Pull request by mkevenaar**
Add documentation about extension package

This allows users to use the extension to install software
from an ISO file.

### CODE REVIEW

**Comment by ferventcoder** [comment]
This looks pretty cool!

File: How-To-Mount-An-Iso-In-Chocolatey-Package.md [hunk 1]
```diff
@@ -16,6 +22,76 @@
 ## Mounting ISOs - The Problem
 There are times when using an installer file directly is not an option as what you need is contained in an ISO. In later versions of the Windows Operating System (defined in [Mount-DiskImage](#mount-diskimage) below), PowerShell provides the ability to mount this ISO file directly using the `Mount-DiskImage` cmdlet.  However, in earlier versions of Windows, this is not possible. In order to maintain backwards compatibility with older Operating Systems, when using an ISO file, you can use ImDisk Virtual Disk Driver.
 
+## chocolatey-isomount.extension
+One of the community maintainers has created an extension package to automate the installation of software from inside an ISO file. This extension is based on the [Mount-DiskImage](#mount-diskimage) method below and therefor has the same [Requirements](#requirements).
```
Comment (gep13) [inline How-To-Mount-An-Iso-In-Chocolatey-Package.md]:
```suggestion
One of the community maintainers has created an extension package to automate the installation of software from inside an ISO file. This extension is based on the [Mount-DiskImage](#mount-diskimage) method below and therefore has the same [Requirements](#requirements).
```

File: How-To-Mount-An-Iso-In-Chocolatey-Package.md [hunk 2]
```diff
@@ -16,6 +22,76 @@
...
+
+### Step 1: Get chocolatey-isomount.extension Package
+You will need to take a dependency on the [chocolatey-isomount.extension package](https://chocolatey.org/packages/chocolatey-isomount.extension) If you are using Chocolatey in an organizational context, be sure to cache the chocolatey-isomount.extension package and place it on your internal sources.
+
+1. MSP/C4B: Run `choco download chocolatey-isomount.extension`
+1. FOSS: Download [imdisk](https://chocolatey.org/api/v2/package/chocolatey-isomount.extension)
```
Comment (gep13) [inline How-To-Mount-An-Iso-In-Chocolatey-Package.md]:
```suggestion
1. FOSS: Download [chocolatey-isomount.extension](https://chocolatey.org/api/v2/package/chocolatey-isomount.extension)
```

**Comment by mkevenaar** [comment]
@gep13 have updated the files with your suggested changes.

**Comment by gep13** [comment]
@mkevenaar your changes have been merged, thanks for your contribution 👍

**Pull request merged**